README.md

```diff
@@ -19,10 +19,16 @@ Requires about 3.5GB of memory to load large Cisco native models.
 
 ## YANG Suite Resources
 
-Landing page: [developer.cisco.com/yangsuite](https://developer.cisco.com/yangsuite)
+DevNet landing page: [developer.cisco.com/yangsuite](https://developer.cisco.com/yangsuite)
 
-Documentatoin: [developer.cisco.com/docs/yangsuite](https://developer.cisco.com/docs/yangsuite)
+Documentation: [developer.cisco.com/docs/yangsuite](https://developer.cisco.com/docs/yangsuite)
 
+## Prerequisites
+
+To use the resources in this repository, you must install Docker on the system where you want to 
+run YANG Suite. Go to https://docs.docker.com/get-docker/ for downloads. 
```

Comment:
The docker-compose install includes a docker install.  You don't need to install docker then docker-compose.

Reply:
Makes sense, I've removed that "Get Docker" portion.

Reply:
Jeremy says you have to install both for Ubuntu Linux.  Has been a long time since I have done a Linux install so did not recall that.  For Mac and Windows the docker-compose install adds both docker and docker-compose.

Who wanted to make this change?  I purposely left out these details so the user would look up the most recent recommended installs for docker according to their OS.  I would make a bet that some versions of Linux have one install package and if not, probably will see that in the future.

Reply:
We have had a few questions (more than a few) in the DevNet Support rooms online, I'm hoping to clarify that there are prerequisites.

Reply:
If you point users to the link you are proposing the Linux installs will not have docker-compose.

Below is the first paragraph at the docker-compose install link that is already in the README.  If you want to update the README, add similar text rather than just pointing users to the docker engine install.

### Prerequisites
Docker Compose relies on Docker Engine for any meaningful work, so make sure you have Docker Engine installed either locally or remote, depending on your setup.

On desktop systems like Docker Desktop for Mac and Windows, Docker Compose is included as part of those desktop installs.

On Linux systems, first install the Docker Engine for your OS as described on the Get Docker page, then come back here for instructions on installing Compose on Linux systems.

To run Compose as a non-root user, see Manage Docker as a non-root user.